Dockerfile

```diff
@@ -47,20 +47,20 @@ RUN apk add --no-cache curl
 SHELL ["/bin/ash", "-eo", "pipefail", "-c"]
 RUN curl -LO "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl" \
     && chmod +x ./kubectl && mv ./kubectl /usr/local/bin/kubectl \
-    && curl -sSfL https://github.com/calyptia/cli/releases/download/v0.41.1/cli_0.41.1_linux_amd64.tar.gz | tar -xz \
+    && curl -sSfL https://github.com/calyptia/cli/releases/download/v1.4.9/cli_1.4.9_linux_amd64.tar.gz | tar -xz \
     && chmod +x ./calyptia && mv ./calyptia /usr/local/bin/calyptia \
     && mkdir -p /linux \
     && cp /usr/local/bin/kubectl /linux/ \
     && cp /usr/local/bin/calyptia /linux/
 
 RUN curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/darwin/amd64/kubectl" \
-    && curl -sSfL https://github.com/calyptia/cli/releases/download/v0.41.1/cli_0.41.1_darwin_amd64.tar.gz | tar -xz \
+    && curl -sSfL https://github.com/calyptia/cli/releases/download/v1.4.9/cli_1.4.9_darwin_amd64.tar.gz | tar -xz \
```

Comment:
Probably bump these up to 1.6.1 now to handle the internal/external cluster URL and other fixes

Reply:
Done